.github/workflows/rhel-9-3.yml

```diff
@@ -782,3 +782,88 @@ jobs:
           path: |
             *.json
             *.log
+
+  pre-edge-vsphere-93:
+    needs: pr-info
+    if: ${{ needs.pr-info.outputs.allowed_user == 'true' && github.event.issue.pull_request &&
+            (endsWith(github.event.comment.body, '/test-rhel-9-3') ||
+            endsWith(github.event.comment.body, '/test-rhel-9-3-vsphere')) }}
```

Comment:
Please add `/test-rhel-9-3-vsphere` at the start of this workflow.

Reply:
added